media-gfx/img2pdf/img2pdf-0.3.4.ebuild

```diff
@@ -14,13 +15,11 @@ SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
 LICENSE="LGPL-3+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="test"
-RESTRICT="!test? ( test )"
 
-DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
-	test? ( dev-python/pdfrw[${PYTHON_USEDEP}] )"
+BDEPEND="test? (
+		dev-python/pdfrw[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP},tiff,zlib]
```

Comment:
det adds RDEPEND to BDEPEND-test?.

Reply:
Not sure if I follow. BDEPEND and RDEPEND are different because tiff and zlib are required for the tests but optional at runtime (depending on what input formats you care about).

Reply:
Ah sorry, my eyes deceived me.